docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -74,6 +74,12 @@
             <td>Boolean</td>
             <td>Whether to generate -U, +U changelog for the same record. This configuration is only valid for the changelog-producer is lookup or full-compaction.</td>
         </tr>
+        <tr>
+            <td><h5>changelog-producer.row-deduplicate-ignore-sequence-field</h5></td>
```

Comment:
Can we support `changelog-producer.row-deduplicate-ignore-fields` here? In some scene, we may want to ignore some unimportant field's change to avoid produce changelog not only sequence filed.

Reply:
Thanks for your suggestion, supporting changelog-producer.row-deduplicate.ignore-fields can indeed cover more scenes, I will change it.

Reply:
Fixed